app/displays.py

```diff
@@ -0,0 +1,35 @@
+from abc import ABC, abstractmethod
+
+
+class Display(ABC):
+    @staticmethod
```

Comment:
The primary issue with staticmethod is that it doesn't allow for polymorphism, which is central to object-oriented programming. If serialize is a static method, you can't override it in the same flexible way with different behaviors for different subclasses. With a normal method (non-static), you can rely on instances to determine which subclass's serialize method to call dynamically.